site/content/in-dev/unreleased/metastores.md

```diff
@@ -108,5 +108,5 @@ The following shows a sample configuration for integrating Polaris with Postgres
 To build Polaris with the necessary Postgres dependency and start the Polaris service, run the following:
 ```bash
 polaris> ./gradlew --no-daemon --info -PeclipseLink=true -PeclipseLinkDeps=org.postgresql:postgresql:42.7.4 clean shadowJar
-polaris> java -jar  polaris-service/build/libs/polaris-service-*.jar server ./polaris-server.yml
+polaris> java -jar dropwizard/service/build/libs/polaris-service-*.jar server ./polaris-server.yml
```

Comment:
Related: should the config file be renamed to `polaris-dropwizard-service.yml`, too?